core/math/vector2.cpp

```diff
@@ -209,3 +210,43 @@ Vector2::operator String() const {
 Vector2::operator Vector2i() const {
 	return Vector2i(x, y);
 }
+
+Rect2 Vector2::fit_keep_aspect_centered(const Vector2 &p_rect) const {
+	const real_t available_ratio = width / height;
+	const real_t fit_ratio = p_rect.width / p_rect.height;
```

Comment:
```suggestion
	real_t available_ratio = width / height;
	real_t fit_ratio = p_rect.width / p_rect.height;
```

Usually we do not use `const` for primitive types, there's no difference for compiler, and it's just adding visual clutter.